encrypted-media-respec.html

```diff
@@ -1928,14 +1912,20 @@ <h5>
                   </li><!-- Invalid input. -->
                   <li>
                     <p>
-                      If <var>content type</var> is not a [=valid media MIME type=] or is
-                      unrecognized, continue to the next iteration.
+                      Let <var>mimeType</var> be the result of running <a>parse a MIME type</a>
+                      with <var>content type</var>.
                     </p>
                   </li>
                   <li>
                     <p>
-                      Let <var>container</var> be the container type specified by <var>content
-                      type</var>.
+                      If <var>mimeType</var> is <code>failure</code> or is unrecognized,
```

Comment:
I was not the author of that phrase, but I think the distinction between "invalid"/"failure" and "unrecognized" is that you could have a valid MIME type that is not recognized as supported for media playback by the UA, e.g. "image/png".

If this is indeed the intent, I think the term "unrecognized" is not the best we could come up with.

How about `If <var>mimeType</var> is <code>failure</code> or is not a supported type for media playback`?  Is that clear?

Reply:
Joey's suggestion is better, but ideally we actually list the MIME types. See also https://mimesniff.spec.whatwg.org/#supported-by-the-user-agent and references.

Reply:
I not convinced that an explicit list of supported media types is practical, given the variability in what is supported across the entire ecosystem of media streaming devices.

Reply:
There's a related PR for Media Capabilities, see "Check MIME Type Support" in https://github.com/w3c/media-capabilities/pull/222/files. We may be able to reuse this, or maybe even just reference it. But in that PR we reference https://mimesniff.spec.whatwg.org/#supported-by-the-user-agent rather than list MIME types.

Reply:
I think "supported by the user agent" plus limiting it to media in some way is reasonable. It will at least give us a central place to work on improving this longer term.